frontend/packages/console-app/locales/OWNERS

```diff
@@ -1,2 +1,26 @@
+reviewers:
+  - dtaylor113
+  - jcaianirh
+  - jeff-phillips-18
+  - jhadvig
+  - kyoto
+  - rhamilto
+  - spadgett
+  - TheRealJon
+  - zherman0
+approvers:
```

Comment:
We don't need to add everyone to the reviewers/approvers here. Approvers are inherited from the parent OWNERS file. I would add @rebeccaalpert as a review here and just yourself as an approver.